Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -92,12 +92,13 @@ void FiniteDifferenceSolver::CalculateCurrentAmpereCylindrical (
         Array4<Real> const& Bt = Bfield[1]->array(mfi);
         Array4<Real> const& Bz = Bfield[2]->array(mfi);
 
-        amrex::Array4<amrex::Real> lr, lt, lz;
-
+        // Extract structures indicating where the fields
+        // should be updated, given the position of the embedded boundaries
```

Comment:
It would be worth it here to note that the plasma current values are calculated at the same location as the E-field, which is why the `eb_update_E` multifab is used. Something like:
```suggestion
        // should be updated, given the position of the embedded boundaries.
        // The plasma current is stored at the same locations as the E-field,
        // therefore the `eb_update_E` multifab also appropriately specifies
        // where the plasma current should be calculated.
```